static/app/types/profiling.d.ts

```diff
@@ -3,7 +3,7 @@ declare namespace Profiling {
   type SentrySampledProfileSample = {
     stack_id: number;
     thread_id: string;
-    elapsed_since_start_ns: string;
```

Comment:
Do we need to handle these with BigInt now?
https://stackoverflow.com/questions/9643626/does-javascript-support-64-bit-integers

Reply:
Doesn't seem necessary at this time because we have 53 bits of precision which is roughly 104 days.

```
>>> 2**53 / 1e9 / 60 / 60 / 24
104.24999137431705
```

Reply:
Should be enough then :D I wonder if we'll ever see this throw at some point in the future :D

Reply:
We'll keep an eye on it, but I don't think we'll have any issues yet